calico/networking/ip-autodetection.md

```diff
@@ -50,39 +50,105 @@ By default, {{site.prodname}} uses the **first-found** method; the first valid I
 - Address used by the node to reach a particular IP or domain (**can-reach**)
 - Regex to include matching interfaces (**interface**)
 - Regex to exclude matching interfaces (**skip-interface**)
-- Address assigned to kubernetes node (**kubernetes-internal-ip**)
+- A list of IP ranges in CIDR format to determine valid IP addresses on the node to choose from (**cidrs**)
+- Address assigned to Kubernetes node (**kubernetes-internal-ip**)
 
 For details on autodetection methods, see [node configuration]({{ site.baseurl }}/reference/node/configuration#ip-autodetection-methods) reference.
 
-#### Manually configure IP address and subnet
+### How to
 
-There are two ways to manually configure an IP address and subnet:
+- [Change the autodetection method](#change-the-autodetection-method)
+- [Manually configure IP address and subnet for a node](#manually-configure-ip-address-and-subnet-for-a-node)
 
-- {{site.prodname}} node container (start/restart)
-  Use environment variables to set values for nodes.
+#### Change the autodetection method
 
-- {{site.prodname}} node resource
-  Update the node resource.
+{% tabs %}
+  <label:Operator,active:true>
+<%
+As noted previously, the default autodetection method is **first valid interface found** (first-found). To use a different autodetection method, edit the default [Installation]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.Installation) custom resource, specifying the method. Below are examples of the supported autodetection methods:
 
-##### Using environment variables and node resource
+> **Note**: To configure the default autodetection method for IPv6 for any of the below methods, use the field `nodeAddressAutodetectionV6`.
+{: .alert .alert-info}
 
-Because you can configure IP address and subnet using either environment variables or node resource, the following table describes how values are synchronized.
+- **IP or domain name**
```

Comment:
I'd call this "Source address used to reach an IP or domain name" - this isn't specifying an address to use, rather it's telling the node "Use the IP selected by the kernel when routing to the given address"